### PR TITLE
Geolocation app

### LINKED ISSUE #55: Implement HTML Geolocator to ask the user for their location

### PR DESCRIPTION
**Pull request by bretm9**
### Participating Group Members:

- @bretm9 

### Code Highlights:

- App.js:
  - Added `showPosition` method that gets user's location via the HTML geolocation API
  - Added `mounted` lifecycle hook that invokes `showPosition`

### Have Tests Been Added?

- [ ] No.
- [ ] Yes, but all tests are not passing.
- [x] Yes, and all are passing.

### Any background context you want to provide?
N/A

### Message/Questions for reviewer:
N/A

### Issues:

- Addresses #
- Closes #55 

### Screenshots (if appropriate):
N/A
### Tracking Consistency:

- [x] added appropriate labels
- [x] My code follows the code style of this project and has removed all unnecessary annotations
- [x] I have added comments on my pull request, particularly in hard-to-understand areas
- [x] All new and existing tests passed
- [x] looked at PR preview to check spelling, syntax, formatting, and completion

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -14,7 +14,8 @@ export default {
   components: { TheHeader },
   data() {
     return {
-      savedParks: []
+      savedParks: [],
+      geolocation: ''
```
Comment (nicolegooden) [inline src/App.vue:18]: Thanks for getting `geolocation` in App.

**Review by nicolegooden — APPROVED** [review]
Awesome additions - appreciate your effort in understanding geolocation and sharing what you've learned with everyone in our meeting this morning!

**Review by npdarrington — APPROVED** [review]
Thanks for implementing Geolocation!

File: src/App.vue (hunk 1 above)
Comment (stacyp2006) [reply src/App.vue:18]: I'm excited to see this in action with our new endpoint

**Review by stacyp2006 — APPROVED** [review]
Great progress! Thanks for taking the lead on this.

**Pull request merged**